README.md

```diff
@@ -3,8 +3,8 @@ kurobako
 
 [![kurobako](https://img.shields.io/crates/v/kurobako.svg)](https://crates.io/crates/kurobako)
 [![Documentation](https://docs.rs/kurobako/badge.svg)](https://docs.rs/kurobako)
-[![Actions Status](https://github.com/sile/kurobako/workflows/CI/badge.svg)](https://github.com/sile/kurobako/actions)
-[![Coverage Status](https://coveralls.io/repos/github/sile/kurobako/badge.svg?branch=master)](https://coveralls.io/github/sile/kurobako?branch=master)
+[![Actions Status](https://github.com/optuna/kurobako/workflows/CI/badge.svg)](https://github.com/optuna/kurobako/actions)
+[![Coverage Status](https://coveralls.io/repos/github/optuna/kurobako/badge.svg?branch=master)](https://coveralls.io/github/optuna/kurobako?branch=master)
 [![License: MIT](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE)
 [![Docker Hub](https://img.shields.io/badge/docker-ready-blue.svg)](https://hub.docker.com/r/sile/kurobako)
```

Comment:
Note that I keep the link untouched in this PR as the docker hub entry "optuna/kurobako" doesn't exist now.

Reply:
Let's track it here https://github.com/optuna/kurobako/issues/40.